client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -33,12 +35,19 @@ interface Props {
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
 
+  const [demoUser, setDemoUser] = useState<Record<string, string> | undefined>();
```

Comment:
We don't a state to store the demo user's info - we can simplify the implementation by storing the demo user in an object :)